nzshm_model/logic_tree/logic_tree_base.py

```diff
@@ -130,39 +110,70 @@ def combined_branches(self) -> Generator[CompositeBranch, None, None]:
                 combined_branch.weight = reduce(mul, weights, 1.0)
             yield combined_branch
 
+    # @classmethod
+    # @abstractmethod
+    # def from_user_config(cls: Type[LogicTreeType], config_path: Union[Path, str]) -> LogicTreeType:
+    #     """
+    #     Create a LogicTree object from a config file
+
+    #     Parameters:
+    #         config_path: path to configuration file
+
+    #     Returns:
+    #         logic_tree
+    #     """
+    #     pass
+
     @classmethod
-    @abstractmethod
-    def from_user_config(cls: Type[_TLogicTree], config_path: Union[Path, str]) -> _TLogicTree:
+    def from_json(cls, json_path: Union[Path, str]) -> 'LogicTree':
         """
-        Create a LogicTree object from a config file
+        Create LogicTree object from json file
+
+        See docs/api/logic_tree/source_logic_tree_config_format.md and
+        api/logic_tree/gmcm_logic_tree_config_format.md
 
         Parameters:
-            config_path: path to configuration file
+            json_path: path to json file
 
         Returns:
             logic_tree
         """
-        pass
+        with Path(json_path).open() as jsonfile:
+            data = json.load(jsonfile)
+        return cls.from_dict(data)
 
     @classmethod
-    def from_json(cls, json_path: Union[Path, str]) -> 'LogicTree':
+    def from_dict(cls: Type[LogicTreeType], data: Dict) -> LogicTreeType:
         """
-        Create LogicTree object from json file or string
+        Create LogicTree object from dict. This function does not de-serialze direct dict representation of a LogicTree,
+        instead it works on a simlified representation of correaltations meant for user-built logic trees.
```

Comment:
@chrisdicaprio  typos  `deserialize, simplified` but actually... is this info correct?? 

I thought the intent was that it DOES handle the to_dict() -> from_dict() round trip??

maybe it does (below)